pom.xml

```diff
@@ -2609,6 +2609,28 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
```

Comment:
Actually, after having this plugin, I were unable to perform parallel builds. 

https://cwiki.apache.org/confluence/display/MAVEN/Parallel+builds+in+Maven+3

Reply:
It sounds like this plug in is not thread safe.

Reply:
Oh, I have not tried this with parallel build. Spotbugs looks [non MT-safe](https://github.com/spotbugs/spotbugs/issues/249)

Reply:
Adding spotbugs slows down my local build a lot. Can we hold this?

Reply:
Yea, this slows down 16ish mins and that was my concern at the very first place. Currently, it only affects Maven build though. +1 for holding on for now.

Reply:
let me check the elapsed time on my environment. +1 for holding on for now.